.gitignore

```diff
@@ -1,4 +1,4 @@
-
+/vendor
```

Comment:
I do not think this change should be in this pull request.

Reply:
yeah, that was made for my local dev's convenience ... just removed it